src/hooks/useDetectOs.ts

```diff
@@ -4,11 +4,28 @@ import downloadUrlByOS from '../util/downloadUrlByOS';
 
 export const useDetectOs = () => {
   const [userOS, setUserOS] = useState<UserOS>(UserOS.UNKNOWN);
+  const [bitness, setBitness] = useState<string>('');
```

Comment:
we can remove this since type inference will take care of it
```suggestion
  const [bitness, setBitness] = useState('');
```

Reply:
Fixed in [f647291](https://github.com/nodejs/nodejs.dev/pull/3163/commits/f6472917dc884ec3c659652199bb1bb829feed2f)